src/lseq/ident.rs

```diff
@@ -255,39 +262,55 @@ impl<A: Actor> IdentGen<A> {
     }
 }
 
-#[cfg(test)]
 impl<A: quickcheck::Arbitrary + Actor> quickcheck::Arbitrary for Identifier<A> {
     fn arbitrary<G: quickcheck::Gen>(g: &mut G) -> Identifier<A> {
-        Identifier {
-            path: Vec::<(u64, Option<A>)>::arbitrary(g),
+        let max_depth = 27; // TODO: where does this come from?
```

Comment:
This is a limitation of `pow` with works with 32 bit exponents

Reply:
We can increase this length but then we can't explore the full range of those layers so it's not all that interesting.

Reply:
Got it, bignums would help here too. I think we really need some real world testing to see if these u32 constraints are actually a problem.